test/varity/vcf_to_hgvs_test.clj

```diff
@@ -242,3 +242,19 @@
       (is (thrown? Exception
                    (vcf-variant->protein-hgvs {:chr "chr7", :pos 55191823, :ref "T", :alt "G"}
                                               test-ref-seq-file rgidx))))))
+
+(deftest coding-dna-ref-gene?-test
+  (testing "valid reference genes"
+    (are [transcript] (#'v2h/coding-dna-ref-gene? {:name transcript})
```

Comment:
I recommend using `true?` or `false?` to test a predicate fn strictly because Clojure treats everything except `false` and `nil` as `true`. 

```clj
(are [transcript] (true? (#'v2h/coding-dna-ref-gene? {:name transcript}))
...
(are [transcript] (false? (#'v2h/coding-dna-ref-gene? {:name transcript}))
```